source/model/PadelMatch.mc

```diff
@@ -65,10 +68,24 @@ class PadelMatch {
         if (self.goldenPoint) {
             // mid game
             if (self.matchStatus.getP1Score() != 40) {
+                if (self.matchStatus.getP1Score() == 'A') {
+                    return incP1Game();
+                }
                 self.matchStatus.incP1Score();
                 return false;
             } else {
-                return incP1Game();
+                if (self.matchStatus.getP2Score() == 40) {
+                    self.matchStatus.incTotalOfDeuces();
+
+                    if (self.matchStatus.getTotalOfDeuces() == self.numberOfDeuces) {
```

Comment:
should we find a way to display to the user if the first deuce has been done already or not? I'm not experienced with these rules, but I guess it would be useful to understand if the current point is "game point" or not

Reply:
if not relevant, then we could argue that for amateur golden point we could just use the advantages mode, and the second time we would just click twice in the same team, to simulate golden point no?

Reply:
Normally no, is not needed, can be cool to have a graphics but if you implement this, maybe is better to have a function that add some graphics so every time, in every situation, if the very next point is the game or set point you can emphasize that, but is only graphics.
For the second, yes, before the mod I actually used this way to "simulate" but you know I think is better to have the watch configured to do that, so there's no way to make mistake

Reply:
After two weeks of use, I think is working but yes, is better to signal the golden/killer point, so maybe after the first deuce, instead return to 40-40, because the very next point will be the winning one, maybe is better to wrote G-G or K-K or some similar. What do you think?